Updated and added NGS QC terms.

c-schroeder opened the pull request:
We updated some QC terms for NGS and added some more that facilitate analysis of tumor-normal experiments. Theses additional QC terms complement the terms already available for QC of raw data, mapping and variant calling, The recommended changes during this pull request were implemented.

mwalzer commented on cv/qc-cv.obo:
```diff
@@ -439,6 +439,14 @@ is_a: QC:1000001 ! meta data parameter
 
+[Term]
+id: QC:1000006
+name: linked file
```
In which context would you say is the proposed use of this term?

mwalzer commented:
Can you provide a small summary description of the terms added?

c-schroeder commented:
We updated some QC terms for NGS and added some more that facilitate analysis of tumor-normal experiments. Theses additional QC terms complement the terms already available for QC of raw data, mapping and variant calling, The recommended changes during this pull request were implemented.
